sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/GenericTypeParameterInOut.cs

```diff
@@ -341,6 +336,11 @@ private static void ReportIssue(ITypeParameterSymbol typeParameter, VarianceKind
                     return true;
             }
 
+            if (namedType.IsTupleType())

```

Comment:
This is the actual fix - when a generic argument is used in a tuple, that argument cannot be neither `in` or `out`